java/test/com/thoughtworks/selenium/CSVTest.java

```diff
@@ -22,6 +22,7 @@
 import org.junit.Before;
 import org.junit.Test;
 
+import java.lang.reflect.InvocationTargetException;
```

Comment:
Can you please revert changes to files in the `thoughtworks` package? This is legacy code and we will eventually phase out RC.

Reply:
done